README.md

```diff
@@ -44,7 +47,12 @@ as compressing will transform the body.
 
 `compression()` accepts these properties in the options object. In addition to
 those listed below, [zlib](http://nodejs.org/api/zlib.html) options may be
-passed in to the options object.
+passed in to the options object or
+[brotli](https://nodejs.org/api/zlib.html#zlib_class_brotlioptions) options.
+
+As for *brotli*, a default is set to compression level 4, unless anything else
+is specified. This is a balanced setting with [a very good speed and a very good
+compression ratio](https://expeditedsecurity.com/blog/nginx-brotli/).
```

Comment:
Please choose a different link for this, as this link demonstrates how to use br with a completely different module than this module, which is just going to be confusing.

Reply:
I don't have a different link and honestly don't have the patience to look for one. I did my own benchmarks, I've seen this one who also did benchmarks, and node's examples also use level 4, but the actual default there is an unusable 11. I'm stripping the link.